elide-async/src/main/java/com/yahoo/elide/async/service/AsyncQueryDAO.java

```diff
@@ -32,7 +31,7 @@
      * @param asyncQueryId UUID of the AsyncQuery to be associated with the AsyncQueryResult object
      * @return AsyncQueryResult Object
      */
-    public AsyncQueryResult createAsyncQueryResult(Integer status, String responseBody, AsyncQuery asyncQuery,
+    public AsyncQuery createAsyncQueryResult(Integer status, String responseBody, AsyncQuery asyncQuery,
```

Comment:
this method can be changed to updateResult since we dont persist AsyncQueryResult in a table. We can just pass the AsyncQueryResult to it.